tests/test_actions.py

```diff
@@ -298,16 +299,12 @@ def test_mention_all_errors_in_order(self, mocked_time, Tester):
     def test_describe(self) -> None:
         mock_action = FakeAction()
         mock_action.describe.return_value = "An African or a European swallow?"
+
         assert (
             Eventually(mock_action).describe()
             == "Eventually an African or a European swallow."
         )
 
-    def test_describe_none(self) -> None:
-        assert (
-            Eventually(FakeAction()).describe() == "Eventually something indescribable."
-        )
-
```

Comment:
Sorry, this was more of a "i saw it so i'll fix it" unrelated change for this PR. This is in `TestEventually`.

Since we have a test for `get_additive_description` that makes this same assertion, and we have a test above that ensures `Eventually` uses `get_additive_description` on its contained performable, i didn't think this test case was necessary.